packages/ensnode-sdk/src/api/indexing-status/index.ts

```diff
@@ -3,3 +3,4 @@ export * from "./request";
 export * from "./response";
 export * from "./serialize";
 export * from "./serialized-response";
+export * from "./zod-schemas";
```

Comment:
In case any Zod schema is needed in ENSApi or ENSIndexer, feel free to import from `@ensnode/ensnode-sdk/internal`. If any zod schema is missing, feel free to add relevant imports to `packages/ensnode-sdk/src/internal.ts`.

https://github.com/namehash/ensnode/blob/ec96ff912c010d4623e586dc2b60a22f122a128d/packages/ensnode-sdk/src/internal.ts#L1-L18

Let's avoid exporting zod schemas from ENSNode SDK in other ways than `packages/ensnode-sdk/src/internal.ts` file.

```suggestion
```

Reply:
Ahh good to know! Thank you! 🙏🏻 Will update these